cmd/kubelet/app/options/options.go

```diff
@@ -87,6 +83,7 @@ type KubeletFlags struct {
 // a kubelet. These can either be set via command line or directly.
 type KubeletServer struct {
 	KubeletFlags
+	ContainerRuntimeOptions
```

Comment:
Not in this PR, but very soon we'll have to actually name these sub-objects. It's getting very confusing what comes from where in KubeletServer.